staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -31,6 +33,9 @@ require (
 )
 
 replace (
+	github.com/asaskevich/govalidator => github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
+	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
+	github.com/mitchellh/mapstructure => github.com/mitchellh/mapstructure v1.1.2
```

Comment:
we can't have replace directives in published modules... this means these have to be bumped as well :-/